<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDlg</class>
 <widget class="QDialog" name="LoginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>登陆</string>
   </property>
   <widget class="QLabel" name="useLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>37</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名:</string>
    </property>
    <property name="buddy">
     <cstring>useLineEdit</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="useLineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pwdLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>32</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string> 密码: </string>
    </property>
    <property name="buddy">
     <cstring>pwdLineEdit</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="pwdLineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="okBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>登陆</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
